sklearn/linear_model/logistic.py

```diff
@@ -922,7 +922,7 @@ def _log_reg_scoring_path(X, y, train, test, pos_class=None, Cs=10,
         check_input=False, max_squared_sum=max_squared_sum,
         sample_weight=sample_weight)
 
-    log_reg = LogisticRegression(fit_intercept=fit_intercept)
+    log_reg = LogisticRegression(multi_class=multi_class)
```

Comment:
Why do we no longer pass fit_intercept?

Reply:
Because if I understand correctly, it is a useless parameter as well as all except multi_class: in this particular case I think it is useless because intercept_ is set manually here:
https://github.com/scikit-learn/scikit-learn/blob/46913adf0757d1a6cae3fff0210a973e9d995bac/sklearn/linear_model/logistic.py#L948-L953

Reply:
agreed but I had to read code to check.

@wdevazelhes when doing thing like this that are not obvious at first add a comment for the next person.

Reply:
Sure, I will do so next time. I talked a bit about it here https://github.com/scikit-learn/scikit-learn/issues/8720#issuecomment-405349408, but I agree this is too far from this part of the code, so I should at least talk about it in the PR, and maybe even with an inline comment